Makefile

```diff
@@ -1,143 +1,143 @@
-
-.PHONY: all
-all: smb
-
-.PHONY: update
-update:
-	hack/update-dependencies.sh
-	hack/verify-update.sh
-
-.PHONY: verify
-verify: unit-test
-	hack/verify-all.sh
-
-.PHONY: unit-test
-unit-test:
-	go test -v -race ./pkg/... ./test/utils/credentials
-
-.PHONY: sanity-test
-sanity-test: smb
-	go test -v -timeout=10m ./test/sanity
-
-.PHONY: integration-test
-integration-test: smb
-	go test -v -timeout=10m ./test/integration
-
-.PHONY: e2e-test
-e2e-test:
-	go test -v -timeout=0 ./test/e2e ${GINKGO_FLAGS}
-
-.PHONY: e2e-bootstrap
-e2e-bootstrap: install-helm
-	docker pull $(IMAGE_TAG) || make smb-container push
-ifdef TEST_WINDOWS
-	helm install smb-csi-driver charts/latest/smb-csi-driver --namespace kube-system --wait --timeout=15m -v=5 --debug \
-		--set image.smb.repository=$(REGISTRY)/$(IMAGE_NAME) \
-		--set image.smb.tag=$(IMAGE_VERSION) \
-		--set windows.enabled=true \
-		--set linux.enabled=false \
-		--set controller.replicas=1
-else
-	helm install smb-csi-driver charts/latest/smb-csi-driver --namespace kube-system --wait --timeout=15m -v=5 --debug \
-		--set image.smb.repository=$(REGISTRY)/$(IMAGE_NAME) \
-		--set image.smb.tag=$(IMAGE_VERSION) \
-		--set snapshot.enabled=true
-endif
-
-.PHONY: install-helm
-install-helm:
-	curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
-
-.PHONY: e2e-teardown
-e2e-teardown:
-	helm delete smb-csi-driver --namespace kube-system
-
-.PHONY: smb
-smb:
-	CGO_ENABLED=0 GOOS=linux go build -a -ldflags ${LDFLAGS} -o _output/smbplugin ./pkg/smbplugin
-
-.PHONY: smb-windows
-smb-windows:
-	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/smbplugin.exe ./pkg/smbplugin
-
-.PHONY: container	
-container: smb	
-	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/smbplugin/Dockerfile .
-
-.PHONY: smb-container
-smb-container:
-ifdef CI
-	az acr login --name $(REGISTRY_NAME)
-	make smb smb-windows
-	az acr build --registry $(REGISTRY_NAME) -t $(IMAGE_TAG)-linux-amd64 -f ./pkg/smbplugin/Dockerfile --platform linux .
-	az acr build --registry $(REGISTRY_NAME) -t $(IMAGE_TAG)-windows-1809-amd64 -f ./pkg/smbplugin/Windows.Dockerfile --platform windows .
-	docker manifest create $(IMAGE_TAG) $(IMAGE_TAG)-linux-amd64 $(IMAGE_TAG)-windows-1809-amd64
-	docker manifest inspect $(IMAGE_TAG)
-else
-ifdef TEST_WINDOWS
-	make smb-windows
-	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/smbplugin/Windows.Dockerfile .
-else
-	make smb
-	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/smbplugin/Dockerfile .
-endif
-endif
-
-.PHONY: push
-push:
-ifdef CI
-	docker manifest push --purge $(IMAGE_TAG)
-else
-	docker push $(IMAGE_TAG)
-endif
-
-.PHONY: push-latest
-push-latest:
-	docker tag $(IMAGE_TAG) $(IMAGE_TAG_LATEST)
-	docker push $(IMAGE_TAG_LATEST)
-
-.PHONY: build-push
-build-push: smb-container
-	docker tag $(IMAGE_TAG) $(IMAGE_TAG_LATEST)
-	docker push $(IMAGE_TAG_LATEST)
-
-.PHONY: clean
-clean:
-	go clean -r -x
-	-rm -rf _output
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
have you changed the format of this file?

Reply:
Actually, I don’t know why this  happened. I am using the same makefile as  upstream.